content/blog/2020-03-16-teaching-online-on-short-notice/index.Rmarkdown

```diff
@@ -0,0 +1,215 @@
+    so move your live teaching online instead of trying to record and edit yourself.
+    (If you *are* going to make videos,
+    please read Chen and Rabb's “[A Pattern Language for Screencasting][screencasting] first.)”
+
+1.  *Learn the song, then improvise over it.*
+    Good musicians don't just make things up on the fly:
+    they learn the song and then improvise around it.
+    *Great* musicians listen to their bandmates and their audience while they do this.
+    The best live teaching (whether in person or online) works the same way:
+    make a plan,
+    then vary your delivery depending on who your learners are,
+    what questions they ask,
+    and what you realize as you're speaking.
+
+1.  *Don't teach to empty chairs.*
+    We all rely on small social cues like nods and "mm hms" far more than we realize.
+    Teaching without these is really hard,
+    so choose a subset of your learners—four at least, half a dozen by preference—and
+    have them turn on video so that your hindbrain believes you actually have an audience.
+    This also promotes engagement:
+    after each break,
+    select a different subset of learners,
+    and always direct your questions to the people you can see.
+
+1.  *Start with a tour.*
+    Many features of your video conference tool will *not* be obvious to participants. 
+    Point out where they can toggle the chat,
+    change their participant view,
+    answer polls,
+    and so on.
+
+1.  *Direct traffic.*
+    Tell people where to put things and where to direct their attention.
+    “Put it in the chat,”
+    “add it to the doc,”
+    and “unmute and tell us” will all make the class run much more smoothly.
+
+1.  *Mute early, mute often.*
+    Background noise kills meetings,
+    so mute *everyone* after every speaker
+    and then unmute the next person you've called on.
+    And as [Chelsea Troy pointed out][troy-meeting],
+    in most online meetings,
+    the first person to speak during a pause gets the floor.
+    The result?
+    "If you have something you want to say,
+    you have to stop listening to the person currently speaking
+    and instead focus on when they're gonna pause or finish
+    so you can leap into that nanosecond of silence and be the first to utter something.
+    The format…encourages participants who want to contribute to say more and listen less."
+    The solution is to run a text chat beside the video conference
+    where people signal that they want to speak,
+    and then call on them in turn.
+
+1.  *Call on people by name.*
+    "Any questions?" often produces an awkward silence,
+    and even when it doesn't,
+    it gives fluent extroverts undue air time.
+    Instead,
+    call on people from your selected group by name.
+    (This also helps even out the effects of differing ability:
+    you can throw easy questions at people who are struggling
+    and save the hard ones for people who might otherwise get bored.)
```

Comment:
Worth adding: "If your web conferencing platform doesn't pick up real names of your students automatically, ask them to note their names when they ask questions in the chat window."

Reply:
Good - added.